### PR TITLE
deps: resolve a number of security alerts

### PR DESCRIPTION
**Pull request by jchappelow**
There are a good number of dependabot security alerts we should resolve before beta period.

https://github.com/kwilteam/kwil-db/security/dependabot

Most are false positive, some are not.

### CODE REVIEW

**Review by brennanjl — APPROVED** [review]

**Review by Yaiba — APPROVED** [review]
👍

**Comment by jchappelow** [comment]
fuu I think testcontainers suddenly got MORE flakey, if that's possible.  PLEASE DON'T MERGE

**Comment by jchappelow** [comment]
Reverted the test containers update for now.  It did work in CI, but I saw a new local failure to start a docker container, so I don't feel too confident about it.  Just the security updates now afaik.

File: core/client/example/main.go [hunk 1]
```diff
@@ -232,22 +231,16 @@ action get_all() public view {
...
 	} // kfSchema := astSchema.(*schema.Schema); j, _ := json.Marshal(kfSchema)
 
-	schemaJSON, err := astSchema.ToJSON()
-	if err != nil {
-		return nil, fmt.Errorf("failed to marshal schema: %w", err)
-	}
-
-	var db transactions.Schema
-	return &db, json.Unmarshal(schemaJSON, &db)
+	return parseRes.Schema, nil
```
Comment (jchappelow) [inline core/client/example/main.go:245]: @brennanjl I hadn't updated the example app when you guys reviewed.  Was forced to do it in this PR in order to update it's deps.  I think this is the intended new way to parse kuneiform.

File: go.mod [hunk 2]
```diff
@@ -9,36 +9,37 @@ replace (
...
+	github.com/ethereum/go-ethereum v1.14.3
+	github.com/grpc-ecosystem/go-grpc-middleware/v2 v2.1.0
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.19.1
+	github.com/jackc/pglogrepl v0.0.0-20240307033717-828fbfe908e9
+	github.com/jackc/pgx/v5 v5.5.5
 	github.com/jpillora/backoff v1.0.0
-	github.com/kwilteam/kwil-db/core v0.1.2
-	github.com/kwilteam/kwil-db/parse v0.0.1
+	github.com/kwilteam/kwil-db/core v0.2.0-beta
+	github.com/kwilteam/kwil-db/parse v0.2.0-beta
```
Comment (jchappelow) [inline go.mod:22]: Obviously these tags don't actually exist.  It's more of documentation, with the `replace`s making it work for now.  We can probably make `beta.1` tags on Monday for these modules though.

**Review by Yaiba — APPROVED** [review]

**Pull request merged**